internal/restic/node_xattr_linux.go

```diff
@@ -0,0 +1,16 @@
+package restic
+
+import (
+	"syscall"
+
+	"github.com/pkg/xattr"
+)
+
+//isXattrNoData check if the error is NODATA('no data available')
+//so the caller can decide whether to ignore this error or not
```

Comment:
There should be a space after "//". The compiler interprets some comments without that space specially.